common/culture/cultures/trogg.txt

```diff
@@ -133,5 +133,56 @@
 			{ name = "mercenary_company_band_of_horned_welshmen" }
 		}
 	}
+
+	drogbar = {
+		graphical_cultures = {
+			tibetan_group_coa_gfx
+			indian_building_gfx
+			mongol_clothing_gfx
+			mongol_unit_gfx
+			
+			drogbar_portrait_gfx
```

Comment:
Will traits be added?

Reply:
I was planning on doing trait in another pr

Reply:
> I was planning on doing trait in another pr

Better do it here because this culture may spawn characters without the race trait.

Reply:
> 
> 
> > I was planning on doing trait in another pr
> 
> Better do it here because this culture may spawn characters without the race trait.

Will do

Reply:
Trait added 3353e27